src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(/images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars:last-child {
+  margin: 0;
+}
```

Comment:
yo can remove `margin-right: 4px;` from `star` selector and use `:not(:last-child)` instead